packages/@n8n/nodes-langchain/utils/httpProxyAgent.ts

```diff
@@ -1,11 +1,16 @@
+import { HttpProxyAgent } from 'http-proxy-agent';
 import { HttpsProxyAgent } from 'https-proxy-agent';
+import proxyFromEnv from 'proxy-from-env';
 
-export function getHttpProxyAgent() {
-	const httpProxy =
-		process.env.HTTPS_PROXY ??
-		process.env.https_proxy ??
-		process.env.HTTP_PROXY ??
-		process.env.http_proxy;
+export function getHttpProxyAgent(baseURL?: string) {
+	if (!baseURL) return undefined;
```

Comment:
Should not we return a ProxyAgent if baseUrl is not defined? For nodes like AWSBedrock?